doc/proxy-clients.rst

```diff
@@ -97,12 +97,28 @@ You can also pass some options to the Varnish client:
 * ``tags_header`` (X-Cache-Tags): Allows you to change the HTTP header used for
   tagging. If you change this, make sure to use the correct header name in your
   :doc:`proxy server configuration <proxy-configuration>`;
+* ``tags_purge_header`` (key): Allows you to change the HTTP header used for
```

Comment:
That was what I did at first using diet for both tag header and purge header, but @ddeboer asked me to split it since xkey's doc uses 'xkey' for requests and 'key' for purge in online examples (the header is different in older versions of the doc other places).